### ISSUE TITLE
Template issue

### ISSUE THREAD

**Issue by Eldenroot** [Destroy666x/MyBB-IP-to-Flags#1]
Globally - add {$mybb->user['flag']} to any template it works and it shows a flag

all other are not working :(

**Comment by Destroy666x** [comment]
In which MyBB version? The plugin hasn't been ported to 1.8 yet. Neither finished and released on 1.6 (but worked fine there, except some small issues and no optimisation).

**Comment by Eldenroot** [comment]
I am sorry - 1.8.4 - but it is OK, I will use global code (I just need to show it in postbin). 

One feature request - maybe it would be better to use a custom css for flag images (manually add into global css file) in next version. Anyway thank you very much, nice plugin!

P.S. Which method do you recommend for IP recognizing? Which is the fastest?

**Comment by Eldenroot** [comment]
@Destroy666x - ok, so it is not working only for MaxMind (only global code is working)

If I change to another matching method -> all codes work

**Comment by Destroy666x** [comment]
The global code will show flag for the current user (not the author of the post) and I don't think you want that.

There is already a custom class for flag images: https://github.com/Destroy666x/MyBB-IP-to-Flags/blob/master/upload/inc/plugins/ip_to_flags.php#L302 I might move the HTML to templates but not sure yet.

All methods are currently slow and not recommended on big boards. They have to read huge databases with IPs. I looked for the best way to optimise it but I haven't found anything useful yet.

I'll fix every issue, make it compatible with 1.8+ and try to optimise it when I'll work on it again. But first I have to recover and push a "new" version I worked on from a disk from old computer. Closing until then.
